UI/style.css

```diff
@@ -0,0 +1,113 @@
+body {
+    background: url('img/background.jpg')
+    /*margin: 0;
```

Comment:
Try and remove the comments on the code

```suggestion
    /*margin: 0;
```

Reply:
Got it. Thank you.

Reply:
The bar icon on the right toggles the navbar list items on a smaller screen enhancing the responsiveness of the web page.
![baricon](https://user-images.githubusercontent.com/45138458/48890114-f2997780-ee48-11e8-9ee4-8f6279fbc707.png)
Toggled list items:
![menutoggled](https://user-images.githubusercontent.com/45138458/48890142-08a73800-ee49-11e8-91b6-184ccf755378.png)